python/cudf/cudf/core/column/column.py

```diff
@@ -1771,6 +1772,24 @@ def build_interval_column(
     )
 
 
+def _time_col_replace_nulls(input_col):
```

Comment:
Should we move this into a method of `DatetimeColumn`? Something like `DatetimeColumn.replace_na_with_null()`?

Reply:
That could work, is there a different version of the `replace` function that operates in place though? Otherwise, would we need to use the same logic but then replace the data internally?

Reply:
I think it's fine for the method to return a new column (most methods in cuDF are out-of-place anyway)